src/test/regress/expected/cont_set_agg.out

```diff
@@ -0,0 +1,65 @@
+CREATE CONTINUOUS VIEW test_set_agg0 AS SELECT set_cardinality(set_agg(x::integer)), exact_count_distinct(x) FROM test_set_agg_stream;
```

Comment:
I checked manually by hard-coding a collision, might take a bit to find a murmur collision to use but I can do it.